Adding Sprocket factor

romainbiard opened the pull request:
Adding Sprocket factor to configure the speed of translation of the navbar regarding the scrollview (factor of 2 means that a move of 2 pts in scrollview move the navbar from 1px).

Thank you !
Romain

andreamazz commented on Source/ScrollingNavigationController.swift:
```diff
@@ -87,7 +88,7 @@ open class ScrollingNavigationController: UINavigationController, UIGestureRecog
    - parameter delay: The delay expressed in points that determines the scrolling resistance. Defaults to `0`
    */
-  open func followScrollView(_ scrollableView: UIView, delay: Double = 0) {
+  open func followScrollView(_ scrollableView: UIView, delay: Double = 0, sprocketFactor: Double = 1) {
```
Please add the new parameter to the documentation comment.

andreamazz commented:
Hi @romainbiard 
This looks good, thank you.  
I'm not a big fan of the name of the parameter though :)  
I think it could be made more clear, something like "scrollSpeed" or "scrollRatio"  
What do you think?

andreamazz commented:
Beautiful. Thanks 👍

romainbiard commented:
Hi @andreamazz and thank you for your feedback.
I've renamed it to scroolSpeedFactor and added some docs missing from previous commit (Sorry for that).
Great work BTW :)

romainbiard commented:
When do you plan to release a new version on Cocoapod ?

andreamazz commented:
Just finished pushing `3.1.0`.  

Cheers

romainbiard commented:
Excellent ! 👏


> Le 14 déc. 2016 à 16:17, Andrea Mazzini <notifications@github.com> a écrit :
> 
> Just finished pushing 3.1.0.
> 
> Cheers
> 
> —
> You are receiving this because you were mentioned.
> Reply to this email directly, view it on GitHub <https://github.com/andreamazz/AMScrollingNavbar/pull/224#issuecomment-267060324>, or mute the thread <https://github.com/notifications/unsubscribe-auth/AAr7hONyLtj0asL7fztSowKRnnxCn7QVks5rIAhygaJpZM4LM1CJ>.
>
